sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/MetricDataMapper.java

```diff
@@ -151,7 +151,11 @@ public static void updateMetricPointBuilder(
         pointBuilder.setValue(pointDataValue);
         // TODO (heya) why give it the same name as otel metric?
         //  it seems this field doesn't matter and only _MS.MetricId property matters?
-        pointBuilder.setName(metricData.getName());
+        if (metricData.getInstrumentationScopeInfo().getName().equals("jmx")) {
+            pointBuilder.setName(metricData.getResource().getSchemaUrl());
+        } else {
+            pointBuilder.setName(metricData.getName());
+        }
```

Comment:
I'm a bit unclear what you mean - the schema url I have at the moment is just the name of the metric as specified in applicationinsights.json or the default jmx metric names that are set in ConfigurationBuilder.java. Do you mean to say that instead of checking the instrumentation scope name, I should check if the schema url starts with some common prefix (in which case, what is that prefix that it should start with?)